packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import Box from '~components/Box';
+import { useDropdown } from '~components/Dropdown/useDropdown';
+import { Text } from '~components/Typography';
+import { getPlatformType, makeAccessible } from '~utils';
+
+type ActionListItemProps = {
+  title: string;
+  value: string;
+  href?: string;
+  /**
+   * Internally passed from ActionList. No need to pass it explicitly
+   */
+  index?: number;
+};
+const ActionListItem = (props: ActionListItemProps): JSX.Element => {
+  const {
+    activeIndex,
+    dropdownBaseId,
+    onOptionClick,
+    selectedIndices,
+    setShouldIgnoreBlur,
+    selectInputRef,
+  } = useDropdown();
+
+  const platformType = getPlatformType();
+  const renderOnWebAs = props.href ? 'a' : 'button';
+  const isReactNative = platformType === 'react-native';
+
+  return (
+    <Box
+      as={!isReactNative ? renderOnWebAs : undefined}
+      id={`${dropdownBaseId}-${props.index}`}
+      role="option"
+      data-value={props.value}
+      data-index={props.index}
+      aria-selected={
+        typeof props.index === 'number' ? selectedIndices.includes(props.index) : undefined
+      }
+      onClick={(e: React.MouseEvent<HTMLButtonElement>) => {
+        if (typeof props.index === 'number') {
+          onOptionClick(e, props.index);
+        }
+      }}
+      onFocus={() => {
+        // We don't want to keep the browser's focus on option item. We move it to selectInput
+        selectInputRef.current?.focus();
```

Comment:
not sure if this shall be true always? 🤔 what happens when we use it for menu?

Reply:
In menu, we shouldn't be using `ActionListItem` in Footer. We have separate `ActionListFooter` and `Button` to handle those scenarios. So this onFocus is never called.